codecov.yml

```diff
@@ -0,0 +1,29 @@
+cat codecov.yml curl --data-binary @- https://codecov.io/validate
```

Comment:
the codecov definition file isn't really required
we need to install it in CI and then run it after the tests finish so it reports back to codecov. you can see an example of how it is used in https://github.com/lirantal/npq/blob/master/.travis.yml as a reference